.circleci/config.yml

```diff
@@ -23,7 +23,7 @@ aliases:
     - checkout
     - run: git submodule update --init --recursive
     - run: swift test
-    - run: brew install swiftlint && swiftlint lint --strict
+    - run: brew install https://raw.github.com/Homebrew/homebrew-core/master/Formula/swiftlint.rb && swiftlint lint --strict
```

Comment:
If you'd prefer you can instead set `HOMEBREW_NO_AUTO_UPDATE=1`

Reply:
That would make it possible to use the old `swiftlint`.

Reply:
Ah yea, you might want to lock this anyways so an update to an existing rule doesn't break PRs unexpectedly

Reply:
Ah, I needed `HOMEBREW_NO_AUTO_UPDATE = 1` for what I intended. Thank you for teaching me.
Duration of `swiftlint` step:
2:00 before: https://circleci.com/gh/jpsim/SourceKitten/3682
6sec after: https://circleci.com/gh/jpsim/SourceKitten/3690

Reply:
> Ah yea, you might want to lock this anyways so an update to an existing rule doesn't break PRs unexpectedly

I think that we are expecting to use the latest swiftlint.